### ISSUE TITLE
iD is creating duplicate nodes.

### ISSUE THREAD

**Issue by drkludge** [openstreetmap/iD#1956]
I cleaned up most of the duplicated nodes world wide on July 14, 2013 12:38 AM  with changeset http://www.openstreetmap.org/browse/changeset/16943483. The duplicate nodes are reported here via OSM Inspector http://tools.geofabrik.de/osmi/?view=geometry&lon=-67.61279&lat=12.20474&zoom=3&overlays=duplicate_node_in_way.  There were a few duplicate nodes left in Asia that I could not clean-up. All off a sudden I see "pimples" all over the world again.  I've cleaned up two of these.  I just cleaned up another one tonight via http://www.openstreetmap.org/browse/changeset/18723423.

Long story short: I believe there is a bug in iD that is creating duplicate nodes that are not the beginning and ending of a polygon.  I've seen the increase of these duplicate nodes after iD was released.  The iD release appears to be after I cleaned up the duplicate nodes.

Here's how to clean up the problem http://wiki.openstreetmap.org/wiki/User:Dr_Kludge/Captain%27s_Logs/osmi/Duplicate_node_in_way.
